angular-css.js

```diff
@@ -597,11 +597,9 @@
               var link = compile ? compile.apply(this, arguments): false;
               return function(scope) {
                 var linkArgs = arguments;
-                $timeout(function () {
-                  if (link) {
-                    link.apply(this, linkArgs);
-                  }
-                });
+                if (link) {
+                  link.apply(this, linkArgs);
+                }
```

Comment:
oh, I think I ran into this same problem a while ago, but doing this caused another bug elsewhere.  Can this change be tested globally?